app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
   <title>RailsStaffGroupProject</title>
+  <script src="https://kit.fontawesome.com/5ca745b1f9.js" crossorigin="anonymous"></script>
```

Comment:
You can look into the font awesome gem rather than the script!
https://github.com/bokmann/font-awesome-rails